clients/lcdproc/main.c

```diff
@@ -36,8 +36,8 @@
 #include "mode.h"
 #include "shared/sockets.h"
 #include "shared/report.h"
-#include "shared/configfile.h"
```

Comment:
Did you remove shared/configfile.c in this PR?

Reply:
Right now, it is still there, but it should be unused and will be removed.